packages/grafana-ui/src/components/Graph/GraphTooltip/SeriesTable.tsx

```diff
@@ -60,9 +65,15 @@ interface SeriesTableProps {
 }
 
 export const SeriesTable: React.FC<SeriesTableProps> = ({ timestamp, series }) => {
+  const theme = useTheme();
+  const styles = getSeriesTableRowStyles(theme);
   return (
     <>
-      {timestamp && <div aria-label="Timestamp">{timestamp}</div>}
+      {timestamp && (
+        <div className={styles.timestamp} aria-label="Timestamp">
+          {timestamp}
+        </div>
+      )}
       {series.map(s => {
```

Comment:
Adds `font-weight`: bold for tooltip's timestamp, to keep the styling of tooltip and context same.